root/var/www/html/freepbx/rest/modules/provisioning-v2.php

```diff
@@ -114,6 +114,76 @@
     return $response->withStatus(500);
 });
 
+$app->post('/provisioning/connectivitycheck', function (Request $request, Response $response, $args) {
+
+    $body = $request->getParsedBody();
+    if (!$body['host']) {
+        return $response->withJson(array('message' => 'missing host parameter'),400);
+    } elseif (!$body['schema']) {
```

Comment:
Please replace everywhere `schema` with `scheme` https://en.wikipedia.org/wiki/URL#Syntax

```suggestion
    } elseif (!$body['scheme']) {
```